app/Cart.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App;
+
+
+class Cart
+{
+    public $items = null;
+    public $totalQty = 0;
+    public $totalPrice = 0;
+
+    public function __construct($oldCart)
```

Comment:
I'm sure this is completely wrong .. why you need to pass an Object in the constructor ?